metapool/sample_sheet.py

```diff
@@ -561,11 +559,35 @@ def quiet_validate_and_scrub_sample_sheet(self):
                 msgs.append(ErrorMessage(f'The {column} column in the Data '
                                          'section is missing'))
 
-        for section in ['Bioinformatics', 'Contact']:
+        # All children of sample_sheet.SampleSheet will have the following four
+        # sections defined: ['Header', 'Reads', 'Settings', 'Data']. All
+        # children of KLSampleSheet will have their columns defined in
+        # child.sections. We will test only for the difference between these
+        # two sets.
+        for section in set(type(self).sections).difference({'Header',
```

Comment:
should this set be a variable? what about global?

Reply:
Also, what are the expected differences between this set and _BIOINFORMATICS_COLUMNS? What about the resulting difference in the for loop?

Reply:
In this case I believe it's fine to have {'Header', 'Reads', 'Settings', 'Data'} defined locally because they are the stock Sections in any base sample-sheet and that's not going to ever change. Also, we're not presently using this set anywhere else.

There currently isn't a sample-sheet variant that adds a new section beyond Bioinformatics and Contact, but it's not inconceivable that one won't in the future. Hence, it's important to determine the set of section-names beyond the above four at run-time and verify that they exist in the sample-sheet.